server/src/schema.js

```diff
@@ -141,6 +141,10 @@ export const Schema = [
     groupId: Int!
   }
 
+  input sendTestPushInput {
```

Comment:
why does this start with lower case when every other input starts with upper case?